classes/Datasets/Transforms.sc

```diff
@@ -0,0 +1,122 @@
+/*****************************************
+Data Transforms
+(C) 2018 Jonathan Reus
```

Comment:
should this be GPL?

Reply:
Should also add Jonathan Reus as a contributor in the README.